shared/src/test/data/formatting-render/yaml/expected/general.yaml

```diff
@@ -1,8 +1,8 @@
 # %Dialect 1.0
 key: val
-multiLine:  |
-  this is a multi line string
-  testing
+multiLine: |
+  this is a multi line
```

Comment:
Whats up with the test change?

Reply:
one of the changes implied erasing an unnecesary space that was added when we had a multiline mark, so I added the mark to test the change.